content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,67 @@
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that can be
+later pulled from to bill your client. This integration provides more control and transparency over
+what the client is charged for.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
 
-.. image:: media/reinvoice_time1.png
+In the *Timesheet on Ticket* drop-down option, choose the Project to which tickets (and timesheets)
+will be linked by default. Open the Project's external link to edit its settings and enable the
+*Billable* option. Save both the Project and the Helpdesk team's settings to apply these features.
+
+.. image:: reinvoice_time/project_billable.png
+   :align: center
+   :alt: Billable feature in Project settings
+
+Create a sales order
+====================
+
+After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Add the customer
+from the ticket to the new quotation. In the *Order Lines* tab, add the *Service on Timesheet*
+product. Click **Confirm** to turn the quotation into a sales order. 
+
+.. note::
+   Remember the :abbr:`SO(Sales Order)` name for later.
+
+.. image:: reinvoice_time/service_on_timesheet.png
    :align: center
-   :alt: Bill from Tasks in Odoo Helpdesk
+   :alt: Add Service on Timesheet product to the SO
 
-Create a sales order and an invoice
-====================================
+Record a timesheet
+==================
 
-Now, once you have recorded the time you spent on the helpdesk ticket, under the *Timesheets* tab,
-access the task clicking on its name.
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking **Tickets** on the
+team's card. You can also access the ticket by going to the Projects dashboard and clicking
+**Tickets** on the Project's card. Click **Edit** and use the *Timesheets* tab to record the time
+you spend on the ticket.
 
-.. image:: media/reinvoice_time2.png
+.. image:: reinvoice_time/record_timesheet_ticket.png
    :align: center
-   :alt: Sales Order from a task in Odoo Helpdesk
+   :alt: Record time spent on a ticket
+
+Link the Helpdesk ticket to the SO
+==================================
 
-*Create Sales Order* and proceed to create the invoice.
+While still on the ticket form, click **Edit** and then click the *Sales Order Item* field. Select
```

Comment:
Change `**Edit**` to `:guilabel:`
Might need to change `*Sales Order Item*` too since it's a field...

Reply:
I changed both `Edit` and `Sales Order Item` to have a `:guilabel:`.